packages/lms/src/lib/components/header/Search.svelte

```diff
@@ -1,25 +1,32 @@
 <script lang="ts">
 	import { goto } from '$app/navigation';
 	import { getSearchResults } from '$lib/utils/fuseSearch/getSearchResults';
-
 	import SearchItem from '../search/SearchItem.svelte';
+	import { parseRawSearchQuery } from '$lib/utils/fuseSearch/parseRawSearchQuery';
 
 	let searchQuery = '';
 	let searchResults: String[] = [];
 	let timer: string | number | NodeJS.Timeout | undefined;
+	let inputFocused = false;
+	let returnedResults = false;
 
 	function handleSubmit(event: { preventDefault: () => void }) {
 		event.preventDefault();
```

Comment:
```suggestion
```
We can instead make use of Svelte's own handling for this by adding `|preventDefault` to the element event, in this case the `form`
```html
<form class="search" on:submit|preventDefault={handleSubmit}>
```